File: Corte_1/Python/Matriz.py
class Matriz:
    def __init__(self):
        self.m = [[0, 0], [0, 0]]

    def datos(self):
        self.m[0][0], self.m[0][1] = 7, 9
        self.m[1][0], self.m[1][1] = 2, 8

    def ver(self):
        for f in self.m:
            for c in f:
                print(c, end=" ")
            print()

    def suma(self):
        s = 0
        for f in self.m:
            for c in f:
                s += c
        return s

# Main
obj = Matriz()
obj.datos()
obj.ver()
print("Suma =", obj.suma())

File: Corte_1/Java/matriz/src/matriz.java
public class matriz {
    public int[][] datos = new int[2][2];

    public void cargarValores() {
        datos[0][0] = 7; datos[0][1] = 9;
        datos[1][0] = 2; datos[1][1] = 8;
    }

    public void mostrarContenido() {
        for (int f = 0; f < 2; f++) {
            for (int c = 0; c < 2; c++) {
                System.out.print(datos[f][c] + " ");
            }
            System.out.println();
        }
    }

    public int calcularTotal() {
        int suma = 0;
        for (int i = 0; i < 2; i++) {
            for (int j = 0; j < 2; j++) {
                suma += datos[i][j];
            }
        }
        return suma;
    }

    public static void main(String[] args) {
        matriz obj = new matriz();
        obj.cargarValores();
        obj.mostrarContenido();
        System.out.println("Resultado de la suma: " + obj.calcularTotal());
    }
}